pages/api/lnurlp/[username].ts

```diff
@@ -63,85 +67,120 @@ const LNURL_INVOICE = gql`
   }
 `
 
+type CreateInvoiceOutput = {
+  paymentRequest?: string
+  error?: Error
+}
+
+const walletUnitCurrency = {
+  BTC: "BTC",
+  USD: "USD",
+} as const
+
+async function createInvoice(
```

Comment:
if createInvoice can only create bitcoin invoices, then it doesn't need to take walletCurrency as an input

Reply:
Yeah. removed the walletCurrency input as well as the object